pkg/print/fs-lvp-velero-config.go

```diff
@@ -101,7 +101,7 @@ data:
 `, red("<private.registry.host>"), lvpTag)
 
 	log.ActionWithoutSpinner("Follow these instructions to set up Velero:\n")
-	log.Info("[1] Install the latest Velero CLI by following these instructions: %s", blue("https://velero.io/docs/v1.6/basic-install/#install-the-cli"))
+	log.Info("[1] Install the latest Velero CLI by following these instructions: %s", blue("https://docs.replicated.com/enterprise/snapshots-velero-cli-installing"))
 	log.Info("[2] Install Velero")
 	log.Info("- For %s, follow these steps:", bold("online installations"))
 	log.Info("	* Create the 'velero' namespace by running the following command:\n\nkubectl create namespace velero")
```

Comment:
There's still a reference to v1.6 docs below this for some of the airgap components.  I know we likely don't want to point to the `"latest"` docs since they made breaking changes in v1.10: https://velero.io/docs/latest/on-premises/#air-gapped-deployments.  Maybe we could bump that to v1.9 for now?  Though that could become outdated if/when we add support for it.

Reply:
I'll update the link to v1.9 for airgap and can add a task to the story there is for adding support for velero v1.10+ to update the link once it's supported.

Reply:
@ajp-io I updated the airgap instructions to link to the latest supported version. The instructions for online installations point to our docs: https://docs.replicated.com/enterprise/snapshots-velero-cli-installing
Should the online instructions link to the latest supported version instead?

Reply:
Yeah, that's what I did in this PR, right? Or let me know if I misunderstood the question. https://github.com/replicatedhq/replicated-docs/pull/909